helm/app/templates/routes/secureRoute.yaml

```diff
@@ -0,0 +1,34 @@
+{{- $route := (lookup "route.openshift.io/v1" "Route" .Release.Namespace "your-route" ) }}
```

Comment:
```suggestion
{{- $route := (lookup "route.openshift.io/v1" "Route" .Release.Namespace "{{ template "ccbc.fullname" . }}-secure" ) }}
```
not sure about the syntax, I hardcoded this in cif...

Reply:
whoopsie, tyvm